src/transforms/hexbin.js

```diff
@@ -0,0 +1,113 @@
+import {sqrt3} from "../symbols.js";
+import {identity, maybeColorChannel, valueof} from "../options.js";
+import {hasOutput, maybeGroup, maybeOutputs, maybeSubgroup} from "./group.js";
+import {initialize} from "./initialize.js";
+
+// We don’t want the hexagons to align with the edges of the plot frame, as that
+// would cause extreme x-values (the upper bound of the default x-scale domain)
+// to be rounded up into a floating bin to the right of the plot. Therefore,
+// rather than centering the origin hexagon around ⟨0,0⟩ in screen coordinates,
+// we offset slightly to ⟨0.5,0⟩. The hexgrid mark uses the same origin.
+export const ox = 0.5, oy = 0;
```

Comment:
These are no longer even used by this file. 😦 It looks like it was removed in 886615b40d8d490d1612914843cfb27f0255731f (as part of #810) but it’s hard to trace since this code has been rewritten so many times. In any case, looks like we’ll need to change the implementation of `hbin` below to actually respect these values, or at the very least, and a big warning comment to the code below that it needs to be consistent with the values of `ox` and `oy` above (if it even is?).